core/src/main/scala/better/files/File.scala

```diff
@@ -224,8 +224,12 @@ class File private(val path: Path)(implicit val fileSystem: FileSystem = path.ge
     * @param attributes
     * @return
     */
-  def createDirectories()(implicit attributes: File.Attributes = File.Attributes.default): this.type = {
-    Files.createDirectories(path, attributes: _*)
+  def createDirectories()(implicit attributes: File.Attributes = File.Attributes.default, linkOptions: File.LinkOptions = File.LinkOptions.default): this.type = {
+    try {
+      Files.createDirectories(path, attributes: _*)
+    } catch {
+      case _: FileAlreadyExistsException if Files.isDirectory(path, linkOptions: _*) =>
```

Comment:
What about the case you mentioned when we want to create the directory with same name but different attributes but the directory already exists but with different attributes. In that case, don't we want an exception here?

Reply:
The `createDirectory()` already throws exception in such case. `createDirectories()` is convenience function. It's not the standard like the underlying system calls. IDK. `mkdir -p` doesn't complain. This seems reasonable for me.